metadata-transformer/src/test/java/no/unit/nva/metadata/service/testdata/LanguageArgumentsProvider.java

```diff
@@ -0,0 +1,25 @@
+package no.unit.nva.metadata.service.testdata;
+
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.ArgumentsProvider;
+
+import java.net.URI;
+import java.util.stream.Stream;
+
+public class LanguageArgumentsProvider implements ArgumentsProvider {
+
+    public static final URI LEXVO_DEU = URI.create("https://lexvo.org/id/iso639-3/deu");
```

Comment:
Certainly a thorough approach. We could perhaps have done without provider classes and split up our tests in separate classes where needed instead, putting this sort of init in a method with @beforeEach or @beforeAll. At that point the use of parameterized tests might fall by the way side.